node/status_node_test.go

```diff
@@ -48,8 +53,11 @@ func TestStatusNodeStart(t *testing.T) {
 	accountManager, err := n.AccountManager()
 	require.Nil(t, err)
 	require.NotNil(t, accountManager)
+	keyStore, err := n.AccountKeyStore()
+	require.Nil(t, err)
```

Comment:
```suggestion
	require.NoError(t, err)
```

Reply:
thanks, I won't apply this as this is part of the revert commit, I believe it will be fixed once we re-apply